jerry-core/config.h

```diff
@@ -79,6 +79,10 @@
 # define JERRY_ESNEXT 1
 #endif /* !defined (JERRY_ESNEXT) */
 
+#ifndef JERRY_BUILTIN_AGGREGATE_ERROR
```

Comment:
I realized something. What happens if this is disabled, but Promise is enabled. Is it builds? Maybe this should be controlled by the Promise builtin, or at least make it dependent from it.

Reply:
True, i removed aggregate builtin macro and replaced it with promise builtin